.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
Hi, I am having issue with npm@7 , I have to add this config for npm to fix the issue
https://twitter.com/ruyadorno/status/1357383518901460993